frontend/app/src/entities/webhook/ui/webhook.spec.ts

```diff
@@ -1,6 +1,6 @@
 import { expect, test } from "@playwright/test";
-import { ACCOUNT_STATE_PATH } from "../../constants";
-import { saveScreenshotForDocs } from "../../utils";
+import { ACCOUNT_STATE_PATH } from "../../../../tests/constants";
```

Comment:
why moving it? E2E tests are global and not necessarily specific to only 1 domain